zgw/documenten-api/src/main/kotlin/com/ritense/portal/documentenapi/client/DocumentApisConfig.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.ritense.portal.documentenapi.client
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "valtimo.zgw.documentenapis")
+class DocumentApisConfig {
+    var documentapis: Map<String, DocumentApiConfig> = mapOf()
+
+    fun getConfig(documentApi: String): DocumentApiConfig {
+        return documentapis[documentApi]!!
+    }
+
+    fun getConfigForDocumentUrl(documentUrl: String): String {
+        return documentapis.filterValues { documentApiConfig -> documentUrl.contains(documentApiConfig.url) }.keys.stream().findFirst().orElseThrow()
```

Comment:
It might be nice to provide a user friendly message in the orElseThrow

Reply:
fixed